chainer/functions/batch_normalization.py

```diff
@@ -91,6 +92,7 @@ class BatchNormalization(function.Function):
     gradient_names = ('ggamma', 'gbeta')
 
     def __init__(self, size, decay=0.9, eps=1e-5):
+        self.size = size
```

Comment:
How about forcing `self.size` to be a tuple? E.g.: `self.size = size if isinstance(size, tuple) else (size,)`?